src/sql/azdata.proposed.d.ts

```diff
@@ -1934,4 +1934,21 @@ declare module 'azdata' {
 			isPrimary: boolean;
 		}
 	}
+
+	export interface ModelBuilder {
+		chart(): ComponentBuilder<ChartComponent, ChartComponentProperties>;
+	}
+
+	export interface ChartComponentProperties extends ComponentProperties {
+		chartType?: string | undefined;
```

Comment:
is it better to use an enum here. Otherwise it will be difficult to discover what types of charts are supported by an extension dev.

Reply:
removed chartType